frontend/package.json

```diff
@@ -23,6 +23,7 @@
   },
   "dependencies": {
     "@next/third-parties": "^14.0.4",
+    "@opentelemetry/api": "^1.8.0",
```

Comment:
fixes a `storybook-build` error

from `.storybook/main.js` , it appears we're disabled from storybook telemetry